pypesto/store/read_from_hdf5.py

```diff
@@ -121,10 +120,9 @@ def read(self, objective: ObjectiveBase = None) -> Problem:
         if objective is None:
             objective = Objective()
             # raise warning that objective is not loaded.
-            warnings.warn(
+            logger.debug(
```

Comment:
Using `warnings` here was intended to make it easier to disable those, see https://github.com/ICB-DCM/pyPESTO/pull/1253